dnf-behave-tests/dnf/steps/fixtures/httpd.py

```diff
@@ -51,7 +51,14 @@ def do_GET(self):
             self.send_response(self.server._conf['status'])
             self.end_headers()
             return
-        super(LoggingHttpHandler, self).do_GET()
+        if self.server._conf.get("response_url"):
+            if self.server._conf["response_url"] == "/{context.dnf.tempdir}/coprs/testuser/testproject/repo/fedora-31" \
+                                                    "/dnf.repo":
+                self.send_response(404)
+                self.send_header("Copr-Error-Data", self.server._conf["headers"]["Copr-Error-Data"])
```

Comment:
"Copr-Error-Data" is also a scenario-specific constant which shouldn't be here, if you do it the way I described, you should get rid of this here.